ui-core/src/main/java/com/adyen/checkout/ui/core/internal/util/ViewExtensions.kt

```diff
@@ -21,7 +21,6 @@ inline var TextInputLayout.isVisible: Boolean
         this.visibility = visibility
         editText?.apply {
             this.visibility = visibility
-            isFocusable = value
```

Comment:
Why was this removed?

Reply:
as if you changed the visibility of the email view more than one time you can't focus anymore on it and it has nothing to do with changing the visibility of the view.

Reply:
It looks like it was added on purpose. @jreij do you remember why this was added?